lib/conjur/fpm/Dockerfile

```diff
@@ -1,14 +1,10 @@
 # Build from the same version of ubuntu as phusion/baseimage
-FROM ubuntu:18.04
+FROM registry.tld/ruby-fips-base-image-phusion:1.0.0
```

Comment:
The appliance image and possum, evoke, ui , ldap-sync packages should compiled on identical env with shared libraries that are in a same location, otherwise it's might not work. also ruby and libpq package were removed from the Dockerfile.
@jvanderhoof

Reply:
it shulde be ubuntu not phusion